src/patterns/menu/Menu.types.ts

```diff
@@ -0,0 +1,26 @@
+import type { Placement } from '@popperjs/core';
```

Comment:
you can import the placement from the popover component to be safe when something on it changes